src/cluster.c

```diff
@@ -1354,15 +1354,17 @@ void addNodeReplyForClusterSlot(client *c, clusterNode *node, int start_slot, in
 
 void clearCachedClusterSlotsResponse(void) {
     for (connTypeForCaching conn_type = CACHE_CONN_TCP; conn_type < CACHE_CONN_TYPE_MAX; conn_type++) {
-        if (server.cached_cluster_slot_info[conn_type]) {
-            sdsfree(server.cached_cluster_slot_info[conn_type]);
-            server.cached_cluster_slot_info[conn_type] = NULL;
+        for (int resp = 0; resp <= 3; resp++) {
```

Comment:
Nit - add "_ver" to improve readability (and in other places too).

```suggestion
        for (int resp_ver = 0; resp_ver <= 3; resp_ver++) {
```

Reply:
Maybe worth mentioning this makes it inconsistent with the variable on the client struct. `resp` is used fairly consistently, although not in that many places.

Reply:
I see - in that way I would also prefer `resp`, for consistency.